input/system/crunchy_bridge/system.go

```diff
@@ -50,6 +43,73 @@ func GetSystemState(ctx context.Context, server *state.Server, logger *util.Logg
 		system.Info.CrunchyBridge.CreatedAt = parsedCreatedAt
 	}
 
+	cpuMetrics, err := client.GetCPUMetrics(ctx)
+	if err != nil {
+		server.SelfTest.MarkCollectionAspectError(state.CollectionAspectSystemStats, "error getting cluster CPU metrics: %s", err)
+		logger.PrintError("CrunchyBridge/System: Encountered error when getting cluster CPU metrics %v\n", err)
+		return
+	}
+	system.CPUStats = make(state.CPUStatisticMap)
+	// In the pganalyze UI, we re-calculate these percentage based on all available percentages
+	// Meaning, even the user only has 10%, it can inflate to 40% if there is no other item that is using lots of %
+	// To avoid this, use idle as "others %" - it might not be correct as there would be other CPU usages, but it at least helps
+	// from showing the wrong percentage for more important ones like system and user
+	cpuIdlePercent := 100 - cpuMetrics.Iowait - cpuMetrics.System - cpuMetrics.User - cpuMetrics.Steal
```

Comment:
Makes sense, I think its reasonable to do this math here to make the graph work as expected.